packages/handlersjs-core/lib/storage/memory-store.ts

```diff
@@ -22,6 +22,7 @@
  * SOFTWARE.
  */
 
+import clone from 'clone';
```

Comment:
@AbelVandenBriel, can you check this library?

Reply:
Library seems fine to me. It has 12M weekly downloads. Takes care of deep cloning so mutability should be fixed. Think this is a good solution.